ansible/playbooks/roles/compatibility_layer/defaults/main.yml

```diff
@@ -18,17 +18,30 @@ gentoo_prefix_path: /cvmfs/{{ cvmfs_repository }}/{{ eessi_version }}/compat/{{
 # How to build the prefix
 prefix_required_space: 15 GB
 prefix_snapshot_url: https://eessi-gentoo-snapshot.s3-eu-west-1.amazonaws.com
-prefix_snapshot_version: 20210607
+prefix_snapshot_version: 20211120
 prefix_user_defined_trusted_dirs:
   - "/cvmfs/{{ cvmfs_repository }}/host_injections/{{ eessi_version }}/compat/{{ eessi_host_os }}/{{ eessi_host_arch }}/lib"
-prefix_singularity_command: "singularity run -B {{ gentoo_prefix_path }}:{{ gentoo_prefix_path }}"
+prefix_mask_packages: |
+  # avoid glibc 2.34, as it's causing issues with the bootstrap, and it's not compatible with CUDA 11.
+  # see https://github.com/EESSI/compatibility-layer/issues/137 + https://bugs.gentoo.org/824482
+  =sys-libs/glibc-2.34
```

Comment:
@bedroge Ugh, we loosened this up a bit too much: the failing version is `glibc-2.34-r2`, so I think we'll need to stick to using `>=` after all here... (but we should be able to keep the `=` for `libgcrypt`)

Reply:
Hmm, okay, changed it back to `>= 2.34`.

Reply:
We have an open issue about this (#137), so we won't easily forget about the masking for glibc 2.34 at least...